container_test.go

```diff
@@ -1426,6 +1426,33 @@ func TestExportContainerNoId(t *testing.T) {
 	}
 }
 
+func TestPutContainerArchive(t *testing.T) {
+	content := "File content"
+	in := stdinMock{bytes.NewBufferString(content)}
+	fakeRT := &FakeRoundTripper{status: http.StatusOK}
+	client := newTestClient(fakeRT)
+	opts := PutContainerArchiveOptions{
+		Path:        "abc",
+		InputStream: in,
+	}
+	err := client.PutContainerArchive("a123456", opts)
+
+	if err != nil {
+		t.Errorf("PutContainerArchive: caugh error %#v while copying from container, expected nil", err.Error())
+	}
+
+	req := fakeRT.requests[0]
+
+	if req.Method != "POST" {
```

Comment:
Should be PUT, no?

The test failed: https://travis-ci.org/fsouza/go-dockerclient/jobs/78424804.